fake/fake-extension.js

```diff
@@ -217,9 +217,9 @@ var seedMvcActions = (function() {
                             { access: 'middleware-start', correlationId: '3', name: 'logger', displayName: 'Morgan Logger', packageName: 'morgan', paths: ['/'] },
                             { access: 'middleware-end', correlationId: '3', name: 'logger', displayName: 'Morgan Logger', packageName: 'morgan', paths: ['/'], result: 'next' },
                             { access: 'middleware-start', correlationId: '4', name: 'jsonParser', displayName: 'JSON Body Parser', packageName: 'body-parser', paths: ['/'] },
-                            { access: 'middleware-end', correlationId: '4', name: 'jsonParser', displayName: 'JSON Body Parser', packageName: 'body-parser', paths: ['/'], result: 'next' },
+                            { access: 'middleware-end', correlationId: '4', name: 'jsonParser', displayName: 'JSON Body Parser', packageName: 'body-parser', paths: ['/'], headers: [{ op: 'set', name: 'content-type', value: 'application/json'}], result: 'next' },
                             { access: 'middleware-start', correlationId: '5', name: 'urlencodedParser', displayName: 'URL-Encoded Body Parser', packageName: 'body-parser', paths: ['/'] },
-                            { access: 'middleware-end', correlationId: '5', name: 'urlencodedParser', displayName: 'URL-Encoded Body Parser', packageName: 'body-parser', paths: ['/'], result: 'next' },
+                            { access: 'middleware-end', correlationId: '5', name: 'urlencodedParser', displayName: 'URL-Encoded Body Parser', packageName: 'body-parser', paths: ['/'], headers: [{ op: 'set', name: 'content-type', value: 'application/xml'}], result: 'next' },
```

Comment:
I can see that... At the same time we will want to keep an eye on ending up with a root level property for every type of change. Maybe we could have a root property call `changes` or `sideeffects` or something that could contain properties for things like headers, status codes, body, cookies, etc. All that said, totally no impact for this PR.